src/sidebar/store/create-store.js

```diff
@@ -174,15 +220,36 @@ export function createStore(modules, initArgs = [], middleware = []) {
     reducer = (state, action) => immutable(originalReducer(state, action));
   }
 
-  // Create the store.
   const store = redux.createStore(reducer, initialState, enhancer);
 
-  // Add actions and selectors as methods to the store.
-  const actions = Object.assign({}, ...modules.map(m => m.actionCreators));
-  const boundActions = redux.bindActionCreators(actions, store.dispatch);
-  const boundSelectors = bindSelectors(allSelectors, store.getState);
+  // Add action creators as methods to the store.
+  /** @type {Record<string, (...args: any[]) => redux.Action>} */
+  const actionCreators = {};
+  for (let module of modules) {
+    assignOnce(actionCreators, module.actionCreators);
```

Comment:
Previously there was no check for two different modules defining conflicting actions. Now there is.